cmd/upgrade/roles/cmd.go

```diff
@@ -458,6 +474,21 @@ func run(cmd *cobra.Command, argv []string) {
 			)
 		}
 	}
+
+	if isUpgradeNeedForAccountRolePolicies && mode == interactive.ModeAuto || isOperatorPolicyUpgradeNeeded {
+		newRolePolicyBindings, err := ocmClient.ListRolePolicyBindings(cluster.ID(), true)
+		if err != nil {
+			reporter.Warnf("Failed to get rolePolicyBindings after upgrade," +
+				" please check the attached policies on upgraded roles")
```

Comment:
I think we need to be clearer on this, what is the desired state? Is it related only to the extra policies?

Reply:
Since it was unable to retrieve the list of role policy bindings, rosa cannot confirm on behalf of the customer if any of the initial arbitrary policies were dropped. 

The desired state in this case would be what the customer intends to have on each role.

Would a better message be,
"Please ensure that the list of policies attached to roles meet your requirements?"

Reply:
I changed to "please ensure the required policies attached on upgraded roles", is this ok? @davidleerh @gdbranco

Reply:
"Please ensure that the required policies are attached to the upgraded roles."
Thank you :)

Reply:
updated the messages